package.json

```diff
@@ -41,19 +41,19 @@
     "he": "^1.1.0",
     "lodash": "^4.11.1",
     "normalize-url": "^1.5.3",
-    "p-queue": "^1.1.0",
+    "p-queue": "^2.1.0",
```

Comment:
Seems it doesn't break anything, but just to double check this

/cc @s0ph1e

Reply:
I've checked, looks good